tracer/test/snapshots/AzureFunctionsTests.InProcess.verified.txt

```diff
@@ -4,7 +4,7 @@
     SpanId: Id_2,
     Name: azure_functions.invoke,
     Resource: Timer TriggerAllTimer,
-    Service: AzureFunctionsAllTriggers,
+    Service: azurefunctionsalltriggers,
```

Comment:
Not sure why these snapshots got updated, but the new values looks like what the agent would normalize the service name and operation name into (lowercase and replace spaces with underscores).

Reply:
Oh, I know. We are enabling `DD_TRACE_STATS_COMPUTATION_ENABLED` which does the same normalization as the agent before computing stats. I need to remove that since we won't use it anymore. It was only used during the preview with the "mini-agent."

Reply:
I removed the code that enables `DD_TRACE_STATS_COMPUTATION_ENABLED` for Azure Functions and GCP and reverted the snapshot changes.